mvpa2/base/config.py

```diff
@@ -225,7 +225,7 @@ def get_as_dtype(self, section, option, dtype, default=None):
             return default
         try:
             return SafeConfigParser._get(self, section, dtype, option)
-        except ValueError, e:
+        except ValueError as e:
             # provide somewhat descriptive error
             raise ValueError, \
```

Comment:
While at it, this statement could also be fixed right away to `raise ValueError("....")`